modules/effects/spec/effect_creator.spec.ts

```diff
@@ -46,6 +46,77 @@ describe('createEffect()', () => {
     );
   });
 
+  it('should create a non-functional effect by default', () => {
+    const obs$ = of({ type: 'a' });
+    const effect = createEffect(() => obs$);
+
+    expect(effect).toBe(obs$);
+    expect(effect['__@ngrx/effects_create__']).toEqual(
+      jasmine.objectContaining({ functional: false })
```

Comment:
I think we we can use Jest's version of this

```suggestion
      expect.objectContaining({ functional: false })
```